doc/source/async_api.rst

```diff
@@ -1,78 +1,94 @@
 Async API (Experimental)
 ========================
 
-Since Python 3.5, it is possible to write concurrent code using the ``async/await`` `syntax <https://docs.python.org/3/library/asyncio.html>`__.
+Since Python 3.5, it is possible to write concurrent code using the
+``async/await`` `syntax <https://docs.python.org/3/library/asyncio.html>`__.
 
-This document describes Ray's support for asyncio, which enables integration with popular async frameworks (e.g., aiohttp, aioredis, etc.) for high performance web and prediction serving.
+This document describes Ray's support for asyncio, which enables integration
+with popular async frameworks (e.g., aiohttp, aioredis, etc.) for highly
+concurrent workloads.
 
+ObjectIDs as asyncio.Futures
+--------------------------
+ObjectIDs can be translated to asyncio.Future. This feature
+make it possible to ``await`` on ray futures in existing concurrent
+applications.
 
-Converting Ray objects into asyncio futures
--------------------------------------------
+Instead of:
 
-Ray object IDs can be converted into asyncio futures with ``ray.experimental.async_api``.
+.. code-block:: python
+
+    @ray.remote
+    def some_task():
+        return 1
+
+    ray.get(some_task.remote())
+    ray.wait([some_task.remote()])
+
+you can do:
+
+.. code-block:: python
+
+    @ray.remote
+    def some_task():
+        return 1
+
+    await some_task.remote()
+    await asyncio.wait([some_task.remote()])
```

Comment:
is there some way to indicate this on the `.remote` interface?

Reply:
ie, in the package reference/docstring?

Reply:
added to ray.get ray.wait docstring